packages/dd-trace/src/plugins/tracing.js

```diff
@@ -12,6 +12,9 @@ class TracingPlugin extends Plugin {
     this.operation = this.constructor.operation
 
     this.addTraceSub('start', message => {
+      if (message && typeof message === 'object') {
+       this.constructor.storesByContext.set(message, storage.getStore())
```

Comment:
```suggestion
        this.constructor.storesByContext.set(message, storage.getStore())
```

Reply:
Can we do this automatically, or at least put it behind a clearer API? Not a fan of all that property access chaining to get at the map.